app/db/models/law_policy/family.py

```diff
@@ -9,11 +12,24 @@
 from .geography import Geography
 
 
-class FamilyCategory(str, enum.Enum):
+class _BaseModelEnum(str, enum.Enum):
     """Family categories as understood in the context of law/policy."""
 
-    EXECUTIVE = "EXECUTIVE"
-    LEGISLATIVE = "LEGISLATIVE"
+    @classmethod
+    def _missing_(cls, value):
+        if isinstance(value, str) and not str.istitle(value):
+            return cls(value.title())
+        raise ValueError(f"{value} is not a valid FamilyCategory")
```

Comment:
This error references `FamilyCategory` in `_BaseModelEnum`

Reply:
sorted